code/modules/mob/living/carbon/slime/items.dm

```diff
@@ -270,7 +270,10 @@
 	G.set_species(golem_type)
 	G.name = G.species.get_random_name()
 	G.real_name = G.name
-	to_chat(G, SPAN_NOTICE("You are a golem. Serve your master, and assist them in completing their goals at any cost."))
+	G.accent = G.species.default_accent
```

Comment:
I feel like this should be set in the golem's set_species instead?

Reply:
i figured it's a bit risky to put it there, set_species is on humans, not on species, so i can't override it, and slapping it in there would cause more trouble than a single line here